src/components/AR/cameras/AR-VPS-camera.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Visual Positioning System Camera.
+ * Currently works only using 8thwall Lightship VPS system. (https://www.8thwall.com/docs/web/#lightship-vps)
+ * 
+ * WLE 20230215 - for this camera to run correctly,
+ * clear the "Project Settings/Editor/serverCOEP" field.
+ * Warning - it will disable the WASM thread support.
+ */
+import { ARSession } from '../AR-session';
+
+import { xr8Provider } from '../frameworks/xr8/xr8-provider';
+import { WorldTracking_XR8 } from '../frameworks/xr8/world-tracking-mode-xr8';
+import { ARCamera } from './AR-Camera';
+
+
+ARSession.registerTrackingProvider(xr8Provider);
+
+const WLEComponentTypeName = 'AR-VPS-camera';
+
+class ARVPSCamera extends ARCamera {
+
+  public static TypeName = WLEComponentTypeName;
+  public static Properties = {
+
+  };
+
+  // WorldTracking_XR8 will check this
+  public readonly UsesVPS = true;
```

Comment:
`usesVPS`,  we use camelCase for attribute. You could also just make that a getter to prevent a JS user to erase it:


```ts
class ... {
    get usesVPS() { return true; }
}
```

Reply:
`readonly` will prevent the user from overwriting it, or?